### DISCUSSION TITLE
Miniconda Installation

### DISCUSSION THREAD

**Discussion by whereisananya** [giswqs/geog-312#29] [Q&A] [answered]
Hi! I installed the Miniconda and its Anaconda prompt (miniconda3) works fine. However, when I went to the command prompt and entered "conda", there was an error message. I am not sure if something is broken. Please help. I have attached the figure showing both.

![Screenshot 2024-08-27 060813](https://github.com/user-attachments/assets/a0b12269-dd6f-43c7-8eef-90bf2df641f1)

**Comment by giswqs** [comment, marked as answer]
I just added some instructions on the website on how to access conda in Windows terminal. 
https://geog-312.gishub.org/book/software/miniconda.html#accessing-conda-in-windows-terminal

**Reply by whereisananya** [reply]
Thank you so much for adding this section. Helped a lot!

**Reply by RachelMba** [reply]
Hi Sir. Just a follow up question on this. I already had conda installed in my system while working on ArcGIS Deep Learning and at that time I followed an instruction that had me specify my system variable Path as NVIDIA computing toolkit. If I change the Path to Python as you have instructed, does this have any effect on my system and the corresponding deep learning toolkit I am working with for other projects? Will this change break anything?

Please Help.
![Screenshot (67)](https://github.com/user-attachments/assets/c5c263b2-eb22-4a08-940a-550d62525a52)

**Comment by RachelMba** [comment]
Hi Sir. Just a follow up question on this. I already had conda installed in my system while working on ArcGIS Deep Learning and at that time I followed an instruction that had me specify my system variable Path as NVIDIA computing toolkit. If I change the Path to Python as you have instructed, does this have any effect on my system and the corresponding deep learning toolkit I am working with for other projects? Will this change break anything? 

Please Help.

![Screenshot (67)](https://github.com/user-attachments/assets/e898e6e6-de10-46de-81fd-160fd5b40704)

**Reply by giswqs** [reply]
No, it should not break anything. If it does, you can always delete the added line in the PATH envronment variable.

**Reply by RachelMba** [reply]
Thank you very much Sir.
 I have been able to get it to work.
